recipes/sncosmo/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sncosmo" %}
+{% set version = "1.4.0" %}
+{% set sha256 = "a26bcb0504213fb095217343a111dd077516d01226e7e1c66e67a8f6ebc352f7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
You don't actually need astropy-helpers, you can just include here ``--offline --no-git``